include/cpp-statsd-client/StatsdClient.hpp

```diff
@@ -78,20 +81,19 @@ class StatsdClient {
                    const uint16_t port,
                    const std::string& prefix,
                    const uint64_t batchsize = 0,
-                   const uint64_t sendInterval = 1000) noexcept;
+                   const uint64_t sendInterval = 1000,
+                   const unsigned int gaugePrecision = 4) noexcept;
 
     //! Returns the error message as an std::string
     const std::string& errorMessage() const noexcept;
 
     //! Increments the key, at a given frequency rate
-    void increment(const std::string& key,
-                   float frequency = 1.0f,
-                   const std::vector<std::string>& tags = {}) const noexcept;
+    void increment(const std::string& key, float frequency = 1.0f, const std::vector<std::string>& tags = {}) const
+        noexcept;
```

Comment:
nothing changed here but there is still a diff, i guess you must have a different clangformat version or a different default line width? any chance you can clean up these whitespace changes so that the diff is just focused on actual changes?